app/models/mixins/ansible_runner_auth_translations.rb

```diff
@@ -0,0 +1,21 @@
+module AnsibleRunnerAuthTranslations
+  AUTH_TYPES = %i[credential vault_credential cloud_credential network_credential]
+
+  module_function
```

Comment:
By making this method a `module_function`, this can be called in two ways:

- By `include`ing it, and calling it as we have done currently (as if it were a method in the class)
- Calling it from it's `module` directly as if it was a class method (similar to `FileUtils`)

The second form would look like:

```ruby
AnsibleRunnerAuthTranslations.translate_credentials(options)
```

Which has the advantage of not adding methods to the classes we are writing this for, though, when adding a module that has `module_function`s, it should add them into the private space (I think...).